reward/reward_distributor.go

```diff
@@ -157,6 +157,12 @@ func GetTotalTxFee(header *types.Header, config *params.ChainConfig) *big.Int {
 	return totalFee
 }
 
+// config.Istanbul must have been set
+func IsRewardSimple(config *params.ChainConfig) bool {
+	policy := config.Istanbul.ProposerPolicy
+	return policy != uint64(istanbul.WeightedRandom)
```

Comment:
I guess this code was meant to check the mainnet. Maybe we can refactor this some day...